package.json

```diff
@@ -80,12 +80,12 @@
     "electron-updater": "^4.3.1",
     "fix-path": "^3.0.0",
     "fs-extra": "^9.0.0",
+    "go-ipfs-dep": "^0.5.1",
```

Comment:
remove ^ please, we don't want to ship versions of go-ipfs that we did not explicitly test :)

Reply:
Nice catch! I missed that 😅